submissions/rafiatAminu/JavaScript/cond.js

```diff
@@ -0,0 +1,25 @@
+let a = 0;
```

Comment:
what's this variable doing? it doesn't have any usage

Reply:
I have taken it out